Source/Core/Models/Event.cs

```diff
@@ -81,6 +81,7 @@ public static class KnownDataKeys {
             public const string SubmissionMethod = "@submission_method";
             public const string SessionEnd = "sessionend";
             public const string SessionHasError = "haserror";
+            public const string ManualStackingKey = "@manual_stacking_key";
```

Comment:
Wondering if we should pick something simpler for this as I want it to feel close to a 1st class thing without actually being a real property on the event. Maybe just "@stack"?